docs/guide/siteaccess.md

```diff
@@ -96,6 +96,28 @@ ezpublish:
 
     A new SiteAccess is recognized by the system, but an Anonymous User will not have read access to it until it is [explicitly given via the Admin > Roles panel](permissions.md#use-cases). Without read access the Anonymous User will simply be directed to the default login page.
 
+### Defining SiteAccess name
+
+In order to simplify the interface and create a better editorial experience, you can "hide"
+ the SiteAccess code and substitute it with a human readable name of the website e.g. `Tastefull Planet`. 
```

Comment:
`human-readable`.
Also, I don't think the SiteAccess name has to be related to the site name, AFAIK it can be anything we want?
Also, `Tasteful` with single L